semeio/jobs/rft/trajectory.py

```diff
@@ -88,6 +93,41 @@ def __getitem__(self, i):
     def __len__(self):
         return len(self.trajectory_points)
 
+    def to_dataframe(self):
+        """Expose the trajectory data as a Pandas DataFrame.
+
+        The data grid_ijk in self is a tuple, this is split into
+        three distinct columns, i, j and k.
+
+        Returns:
+            pd.DataFrame
+        """
+        dframe = pd.DataFrame(data=[vars(point) for point in self.trajectory_points])
+        dframe["is_active"] = [point.is_active() for point in self.trajectory_points]
+        dframe["inactive_info"] = [
+            point.inactive_info() for point in self.trajectory_points
+        ]
+        if "grid_ijk" in dframe:
```

Comment:
Might be good to split this onto a couple of helper functions? Would make it easier to read.

Reply:
`grid_ijk` refactored into its own tuple-splitting-function